jest.setup.js

```diff
@@ -3,6 +3,5 @@
 const { toHaveNoViolations } = require('jest-axe')
 
 jest.setTimeout(10000)
-jest.retryTimes(3, { logErrorsBeforeRetry: true })
```

Comment:
🎉 

I hated putting this in, so very hopeful we've shaken all the flakes out now!